Qa/2.0.0 (#26)
witnet/witnet-solidity: README.md

These are the most relevant features that make the **Witnet Oracle** stand out from other blockchain oracle solutions:

- **#1**: Data requests can be parametrically declared and verified from within smart contracts.
- **#2**: Any sort of data (JSON objects, primitive values or even binary buffers) can be pulled anytime from smart contracts.
- **#3**: Data traceability is possible: actual sources, offchain computations and sampling timestamps can be formally verified, as secured but the whole Witnet Oracle blockchain, in which witnessing nodes get randomly selected for every single data update. 
- **#4**: Data requests are paid with plain EVM currency.
- **#5**: No need to pre-fund smarts contracts with third-party ERC-20 tokens.
- **#6**: No need to undergo offchain subscription workflows.
- **#7**: No need to run your own Witnet nodes, nor any other kind of offchain infrastructure. 

Once you get this package installed and initialized, you will be able to:

- Build and test your own parameterized **Witnet Data Requests**.
- Check the EVM chains currently bridged to the Witnet Oracle Blockchain.

> - Check the number of Witnet data sources, templates and requests already available within your project.

Reviewer: ```suggestion
- Check the number of Witnet data sources, templates and requests that are already available within your project.
```

- Check the EVM addresses of all Witnet-based assets available on the specified chain.
- Deploy your own Witnet-based assets (sources, templates and requests) onto the specified chain. 

> - Run a local ETH/RPC gateway and a Web3 console as to ease both the deployment and the interaction or testing of your Witnet-based assets.

Reviewer: ```suggestion
- Run a local ETH/RPC gateway and a Web3 console to ease both the deployment and the interaction or testing of your Witnet-based assets.
```

> - Run the bundled **Witnet Solidity Wizard**,  as to generate smart contracts skeletons adapted to your actual needs.

Reviewer: ```suggestion
- Run the bundled **Witnet Solidity Wizard** to generate smart contract skeletons adapted to your actual needs.
```

> - Freely read from a selection of **Witnet Price Feeds** that get periodically updated by the Witnet Foundation, being able also to programatically force a price update from within your smart contract, or even introspect the actual providers being used by the Witnet Foundation for every update. 

Reviewer: ```suggestion
- Freely read from a selection of **Witnet Price Feeds** that get periodically updated by the Witnet Foundation, being able also to programmatically force a price update from within your smart contract, or even introspect the actual providers being used by the Witnet Foundation for every update. 
```
